lib/pc_schema.c

```diff
@@ -367,18 +367,6 @@ void pc_schema_check_xyzm(PCSCHEMA *s)
 			continue;
 		}
 	}
-
-	if ( s->x_position < 0 )
-	{
-		pcerror("pc_schema_check_xyzm: invalid x_position '%d'", s->x_position);
-		return;
-	}
-
-	if ( s->y_position < 0 )
-	{
-		pcerror("pc_schema_check_xyzm: invalid y_position '%d'", s->y_position);
-		return;
-	}
```

Comment:
It is redundant with [pc_schema_is_valid](https://github.com/mbredif/pointcloud/blob/4ad548561cc0de82bee4229e7d8693b0e9df7cee/lib/pc_schema.c#L587-L597), which is now [enforced](https://github.com/mbredif/pointcloud/blob/4ad548561cc0de82bee4229e7d8693b0e9df7cee/lib/cunit/cu_pc_schema.c#L161-L173) by pc_schema_from_xml.

This was actually the main trouble for @achidlow, as the unhappy path was heterogeneous : pcerror failure here and PC_FAILURE return value in pc_schema_from_xml.